lightstrip.py

```diff
@@ -0,0 +1,21 @@
+import neopixel
+from patterns import static
+
+class LightStrip:
+
+    def __init__(self, pixel_pin, pixel_count: int) -> None:
+
+        # Create a neopixel for each lightstrip
+        self.neopixel = neopixel.NeoPixel(pixel_pin, pixel_count, brightness=1, auto_write=False)
+        self.PIXEL_COUNT = pixel_count
+
+        # Colors for different patterns
+        self.primary: tuple[int, int, int] = (0, 0, 0)
+        self.secondary: tuple[int, int, int] = (0, 0, 0)
+
+        # Pattern function to call on every update
+        self.pattern = static
```

Comment:
⭐ because it can be confusing using functions as variables, suggest adding a unit to this one so its `pattern_func`